Makefile

```diff
@@ -143,29 +142,26 @@ compile lint test ci : dev-container
 		make docker_$@ ;\
 	fi
 
-upload-container: ## uploads to .  You need to have credentials.  Make sure you set DOCKER_CONFIG=`cd ~/.docker-hub-f4tq/;pwd`
-upload-container: container tag
-	set -x; if [ ! -z "$(PUBLISH_TAG)" ]; then \
-	docker push $(PUBLISH_TAG):`cat VERSION` ; \
-	fi 
-
-tag:  ## tag the image with the provided PUBLISH_TAG 
-tag:
-	@echo "tagging $(PUBLISH_TAG)"
-	@if [ "x$$sha" = "x" ] ; then sha=`git rev-parse HEAD`; fi ;\
-	if [ ! -z "$(PUBLISH_TAG)" ]; then \
-	docker tag mesosphere/etcd-mesos:$$sha $(PUBLISH_TAG):$$(cat VERSION) ; \
-	fi ; \
-	echo "tagged ..."
+
+build-container: ## builds and tags to the current VERSION
+build-container: container tag-container
+
+tag-container: ## Apply docker upstream tags
+tag-container:
+	docker tag adobeplatform/ethos-etcd-mesos:`git rev-parse HEAD` $(PUBLISH_TAG)/ethos-etcd-mesos:`cat VERSION`
+
+#upload-container: ## uploads to adobeplatform.  You need to have credentials.  Make sure you set DOCKER_CONFIG=`cd $$HOME/.docker-hub-f4tq/;pwd`
+upload-container: build-container
```

Comment:
Just so you know, this instance of artifactory does not let you push over existing tags so it's important to make sure you only push once per version.

Reply:
yes, so usually when I test I use `adobeplatform` .  I save artifactory for final publication.  
```
make upload-container DOCKER_CONFIG=`cd ~/.docker-hub-ethos/;pwd` PUBLISH_TAG=adobeplatform
```